app/views/course/admin/announcement_settings/index.html.slim

```diff
@@ -0,0 +1,4 @@
+= simple_form_for @settings, url: course_admin_announcement_path(current_course) do |f|
```

Comment:
More without brackets, so should remove the brackets then.

Feels inconsistent to me, but okay.